posts/models.py

```diff
@@ -17,3 +17,18 @@ def original_or_self(self):
         while p.original_post:
             p = p.original_post
         return p
+
+    def __str__(self):
+        return self.content
+
+class Hashtag(models.Model):
+    name = models.CharField(max_length=255)
+    posts = models.ManyToManyField(Post)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Seems like we don't need this `created_at` timestamp for a hashtag, since `Post` already has the same timestamps.